### DISCUSSION TITLE
[Question]: How to manually ban users

### DISCUSSION THREAD

**Discussion by assassinliujie** [danny-avila/LibreChat#1560] [Q&A] [answered]
### What is your question?

I provide this service for my club and use points to redeem usage permissions on a monthly basis. However, not everyone needs it every month, so in some months, I need to temporarily cancel the usage permissions of some users. What should I do

### More Details

If there are corresponding parameters in SQL that represent the banned user, that would be great

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
There is a simple helper script which you can use to ban users. Using it depends on how you run the app.

If using docker, you need to enter the container shell and run `npm run ban-user`. Otherwise you can run the same command in the project folder, provided you have node/npm installed.
